### DISCUSSION TITLE
Feature Request: Add CLI Commands for FastAPI Similar to `rails generate` and `rails routes`

### DISCUSSION THREAD

**Discussion by sarvesh4396** [fastapi/fastapi-cli#142] [Questions]
### First Check
 - [x] I added a very descriptive title here.
 - [x] I used the GitHub search to find a similar question and didn't find it.
 - [x] I searched in the documentation/README.
 - [x] I already searched in Google "How to do X" and didn't find any information.
 - [x] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help
 - [x]  I commit to help with one of those options 👆

### Example Code
NA

### Description

I recently worked with rails and i am curious that can some of its features can be implemented in `fastapi` it-self
I propose enhancing the FastAPI CLI with developer-friendly commands like:  

1. **`fastapi generate`**  
   Generate boilerplate for models, routes, controllers, etc.  
   Example:  
   ```bash  
   fastapi generate model User --fields "name:str,email:str,is_active:bool"  
   ```  

2. **`fastapi routes`**  
   List all routes in a structured format:  
   ```bash  
   fastapi routes  
   ```
   Output:  
   ```
   Method   Path               Name                 Handler  
   GET      /users             get_users           app.users.get_users  
   POST     /users             create_user         app.users.create_user  
   ```  

### **Why?**  
- Speeds up development.  
- Standardizes app structure.  
- Simplifies route overview and debugging.  

We may Standardize the structure.

### Operating System

Linux, Windows, macOS

### Operating System Details

Windows 10

### Python Version

3.10

**Comment by FlavienRx** [comment]
Hey, 

`fastapi routes` could be useful 👍

**Comment by YuriiMotov** [comment]
Similar:
* https://github.com/fastapi/fastapi-cli/discussions/170
* https://github.com/fastapi/fastapi-cli/discussions/130
